backend/app/controllers/memos_controller.rb

```diff
@@ -45,7 +47,7 @@ def destroy
 
   private
 
-  def memo_params
-    params.require(:memo).permit(:title, :content, :poster)
+  def form_params
+    params.require(:form).permit(:title, :content, :poster, tag_ids: [])
```

Comment:
Idiomatic

memoに関するものなので、memoをtopレベルのkeyとするようにしましょう！

```suggestion
    params.require(:memo).permit(:title, :content, :poster, tag_ids: [])
```